### ISSUE TITLE
getting drunk non stop

### ISSUE THREAD

**Issue by blightrunner** [dalaranwow/dalaran-wow#5742]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 
ogres in outland give you a debuff that gets you drunk, it glitched on me and it only went away after death.
It restarts randomly whenever npc's aply debuffs on me (dazes knock ups stun)
**Expected behaviour**:
the drunk debuff from outland should not come back when npc's from nothrend or dungeons debuff me
**Steps to reproduce the problem**:

1.  get drunk debuff from orges and kill it before debuff runs out ?
2.  hope you get the same strange glich as i do
3. get knocked up dazed stunned etc after glitch is pressent

**Include proofs for this behaviour**
![bugg](https://user-images.githubusercontent.com/51841682/59545544-15dc5f00-8f20-11e9-86eb-20e3b048bb32.JPG)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
